config/ufs/machines/config_machines.xml

```diff
@@ -451,7 +451,77 @@ This allows using a different mpirun command to launch unit tests
       <env name="W3NCO_LIBd">$ENV{NCEPLIBS_DIR}/lib/libw3nco_v2.0.6_d.a</env>
     </environment_variables>
   </machine>
-
+ <machine MACH="gaea">
+    <DESC>
+      port to gaea
+    </DESC>
+    <NODENAME_REGEX>gaea*</NODENAME_REGEX>
+    <OS>CNL</OS>
+    <COMPILERS>cray-intel</COMPILERS>
+    <MPILIBS>mpich</MPILIBS>
+    <PROJECT>esrl</PROJECT>
+    <SAVE_TIMING_DIR> </SAVE_TIMING_DIR>
+    <CIME_OUTPUT_ROOT>$ENV{SCRATCH}/$ENV{USER}/ufs-runs/scratch</CIME_OUTPUT_ROOT>
```

Comment:
Set CIME_OUTPUT_ROOT to $ENV{SCRATCH}/$ENV{USER} 
but then set the other fields to $CIME_OUTPUT_ROOT/whatever
so that the user only needs to change one field.